app/views/admin/creator_settings/_form.html.erb

```diff
@@ -78,3 +78,21 @@
     </div>
   </fieldset>
 </div>
+
+<div class="crayons-field mt-6 align-left">
+  <fieldset aria-describedby="section-description">
+    <legend class="crayons-field__label mb-2">Finally, please agree to the following:</legend>
+    <div>
+      <div class="mb-2">
+        <%= hidden_field_tag :checked_code_of_conduct, 0 %>
+        <%= check_box_tag :checked_code_of_conduct, 1, class: "crayons-field crayons-field--check-box", required: true %>
+        <label for="checked_code_of_conduct">I agree to uphold our <a href="/code-of-conduct">Code of Conduct</a>.</label>
+      </div>
```

Comment:
@juliannatetreault if you inspect the DOM, you will notice that this is what your HTML looks like currently: 

<img width="575" alt="Screenshot 2021-11-19 at 12 03 43" src="https://user-images.githubusercontent.com/2786819/142604345-7d410ad1-e852-4642-aa94-36aa23cda96c.png">

From the screenshot above you will notice that the class or the required attributes are not there. 

The reason that the checkbox was not working as expected is because the `check_box_tag` uses positional arguments. If you look at the example in the [rails docs](https://apidock.com/rails/ActionView/Helpers/FormTagHelper/check_box_tag)

```
check_box_tag(name, value = "1", checked = false, options = {})
```

should you want to provide options like classnames and required attributes, then you have to provide all the previous arguments in their position as well. The default argument comes into use when you don't need to provide any positional arguments after it. 


```suggestion
      <div class="mb-2">
        <%= hidden_field_tag :checked_code_of_conduct, "0" %>
        <%= check_box_tag :checked_code_of_conduct, "1", false, class: "crayons-checkbox", required: true %>
        <label for="checked_code_of_conduct">I agree to uphold our <a href="/code-of-conduct">Code of Conduct</a>.</label>
      </div>
```

After this change above, if you looks at your HTML you should see all the attributes being applied to it.  

<img width="582" alt="Screenshot 2021-11-19 at 12 09 47" src="https://user-images.githubusercontent.com/2786819/142605250-0b92783a-a1ad-49fb-a3b7-fe88c616b63c.png">

One other thing to not that I changed above is to use the correct classname `crayons-checkbox` which I think ties back into the brand styling we do. You'll notice that i changed the `radio` to work as above [in my PR too](https://github.com/forem/forem/pull/15432/files#diff-2b226e5b31f2d1134741137bf024abac92e633c9f76ba80de8bcff3771c6516aR57) because the classname was not showing up for the same reason as above.

Hope this helps! Ping me if anything is unclear 😊

Reply:
A follow up problem:

![coc](https://user-images.githubusercontent.com/2786819/142606267-d57c84a7-a114-49af-8e39-590b2035ad2b.gif)

If you look closely at my `gif` I click the button (you can't actually see this click in the GIF) and the HTML validation for the checkboxes is not allowing me to press finish because some required fields (the checkboxes) are not filled. However, the notice to say that it's required does not show up unless you hover over the checkbox. This will undoubtedly confuse users. I'm not 100% sure how to solve this, maybe @nickytonline can provide a suggestion on this problem?